uploader/base.py

```diff
@@ -1,14 +1,45 @@
 from flask import Flask
 from flask_restful import Api
+from flask_sqlalchemy import SQLAlchemy
+from flask_marshmallow import Marshmallow
+from sqlalchemy import TIMESTAMP, Enum
+import enum
 import os
-
+import datetime 
 
 app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] =  'postgresql+psycopg2://postgres:1234@database:5432/postgres'
 app.config["SECRET_KEY"] = 'your-secret-key'
 app.config["UNPROCESSED_FOLDER"] = os.path.join(os.getcwd(), 'unprocessed_videos')
 app.config["PROCESSED_FOLDER"] = os.path.join(os.getcwd(), 'processed_videos')
 app.config["RESOURCES_FOLDER"] = os.path.join(os.getcwd(), 'res')
 app.config["LOGO_FILE"] = os.path.join(app.config["RESOURCES_FOLDER"], 'maxresdefault.jpg')
 app_context = app.app_context()
 app_context.push()
-api = Api(app)
\ No newline at end of file
+db = SQLAlchemy(app)
+ma = Marshmallow(app)
+api = Api(app)
+
+class Status(enum.Enum):
+    inprogress = 0
```

Comment:
Podemos usar este tipo de estados ?
```suggestion
    Unprocessed = 'UNPROCESSED'
    Processing = 'PROCESSING'
    Processed = 'PROCESSED'
    Incompleted = 'INCOMPLETED'
```

Reply:
cual es la diferencia entre inprogress y unprocesed? No significarian lo mismo? algo asi como created?

Reply:
Como yo lo veo tenemos los siguientes estados para un video 
Unprocessed-> es el archivo de video tal cual lo envía el usuario
Processing-> el archivo de video fue tomado por la cola y esta siendo procesado( lo que significa, recortando el video agrgando las imagenes adelante y atrás etc)
Processed  -> significa que el archivo fue procesado y guardado correctamente en el fichero de archivos.

Incompleted-> hubo algún error mientras se procesaba (deberíamos guardar rl error)

Reply:
Corregido en commit 92072a5c80253e95c6a71c9cbf09be571eb8a59e